tests/e2e/pageobjects/dashboard/UserPreferences.ts

```diff
@@ -14,14 +14,25 @@ import { By } from 'selenium-webdriver';
 import { DriverHelper } from '../../utils/DriverHelper';
 import { Logger } from '../../utils/Logger';
 
+enum GitServices {
+	GITHUB = 'github',
```

Comment:
IMHO we can avoid duplication if use FACTORY_TEST_CONSTANTS > GitProviderType https://github.com/eclipse/che/blob/07ed023e6d80e46c9a1759d7ff25a77d4e9ab535/tests/e2e/constants/FACTORY_TEST_CONSTANTS.ts#L12-L18